src/model/Track.js

```diff
@@ -0,0 +1,32 @@
+const { DEFAULT_RACING_ROUND } = require('../constants/racing-rule.js');
+
+class Track {
+  #endRound;
+
+  #round;
+
+  constructor() {
+    this.#endRound = DEFAULT_RACING_ROUND;
+    this.#round = 1;
+  }
```

Comment:
```suggestion
  #endRound = DEFAULT_RACING_ROUND;
  #round = 1;
```

이렇게도 될 것 같네요 ~!
생성자 함수에서 처리하는 로직이 별도로 없다면 제거해도 될 것 같아요.